definitions/artifacts/api.json

```diff
@@ -45,18 +45,39 @@
     },
     "specs": {
       "type": "object",
-      "additionalProperties": false,
```

Comment:
addlProp false needs to stay in art defs. Its undebuggable from TF otherwise.

Reply:
Gotcha, added it back